Version.txt

```diff
@@ -0,0 +1 @@
+1.2.0
```

Comment:
Perhaps this should be 1.2.2?

Reply:
The version string is set in CMakeLists.txt at line 61. It would be great if those could be unified so that it is not necessary to maintain multiple version strings.

Reply:
CMakeLists.txt line 61 has been changed as follows:
```diff
-  set (P4C_SEM_VERSION_STRING "1.2.0+g202103291035~a69e52")
+  file(READ "Version.txt" P4C_SEM_VERSION_STRING)
```
This change should allow us to maintain a single version string stored in `Version.txt`. By keeping the version string as a separate file we could detect changes that would trigger the workflow run:
```yaml
on:
  push:
    branches:
      - main
    paths:
      - 'Version.txt'
```

Reply:
>Perhaps this should be 1.2.2?

Thanks, I have updated the PR to use `1.2.2`.